langchain/llms/openai.py

```diff
@@ -54,6 +54,8 @@ class BaseOpenAI(BaseLLM, BaseModel):
     """Batch size to use when passing multiple documents to generate."""
     request_timeout: Optional[Union[float, Tuple[float, float]]] = None
     """Timeout for requests to OpenAI completion API. Default is 600 seconds."""
+    logit_bias: Optional[Dict[str, float]] = {}
```

Comment:
to avoid mutable defaults, can we make this `Field(default_factory=dict)`?

Reply:
my bad. push error...

Reply:
fixed